packages/menu/src/Menu/Menu.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import type { TgphComponentProps } from "@telegraph/helpers";
+import { Lucide } from "@telegraph/icon";
+
+import { Menu as TelegraphMenu } from "./Menu";
+
+const meta: Meta = {
+  title: "Components/Menu",
+  component: TelegraphMenu.Root,
+  argTypes: {},
+  args: {},
+};
+
+export default meta;
+
+type Story = StoryObj<TgphComponentProps<typeof TelegraphMenu.Root>>;
+
+export const Default: Story = {
+  render: ({ ...args }) => {
+    return (
```

Comment:
Nit: It'd be nice to have a full example with the menu button as well to open/close it but not necessary to get this shipped unless there are any gotchas there

Reply:
That is going to be a separate component `dropdown-menu`, this component is just the menu contents as it'll be used across a few different component types.

Reply:
Ahh, makes sense!